### ISSUE TITLE
physics.c does not compile

### ISSUE THREAD

**Issue by teuben** [Jeremycouturier/NcorpiON#1]
There are 4 variables (ax,ay,az,dg[3]) that are not declared and prevent a clean compile.

**Comment by Jeremycouturier** [comment]
Yes, those should have been removed before pushing but I forgot. If you pull again they will have dissapeared.

________________________________
From: Peter Teuben ***@***.***>
Sent: Sunday, March 31, 2024 12:26:05 AM
To: Jeremycouturier/NcorpiON ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [Jeremycouturier/NcorpiON] physics.c does not compile (Issue #1)


There are 4 variables (ax,ay,az,dg[3]) that are not declared and prevent a clean compile.

**Comment by Jeremycouturier** [comment]
Thank you for pointing that out. I am currently adding tidal dissipation to NcorpiON so there might be temporary issues ... Sorry about that

________________________________
